metis-validation/metis-validation-rest/src/main/resources/schema/EDM-COMMON-MAIN.xsd

```diff
@@ -682,7 +682,7 @@
         </annotation>
     </element>
 
-    <element name="rights" type="rdf:ResourceType">
+    <element name="rights" type="rdf:RightsStatements">
```

Comment:
Okay this change affects corelib as well, codewise I mean

Reply:
interesting... is ResoruceType a class?

Reply:
Yes, so it will be a different one therefore code changes are required.